### DISCUSSION TITLE
Migration to v4 works on gatsby cloud but not on netlify.

### DISCUSSION THREAD

**Discussion by wad-im** [gatsbyjs/gatsby#34219] [Help] [answered]
Hello,

I updated [Quercus Group](https://quercus-group.com/) Website from v3 to v4 and got immediately failed builds on netlify where I host the page. The error message that I get on netlify is that "Gatsby requires Node.js 14.15.0 or higher (you have v12.18.0).", which is confusing because I am on version 14.15.4. 

I tried on gatsby cloud, and got a failed build there due to some dependency issues. I managed to resolve those and the page build succeeded on gatsby cloud. 

But still not on netlify. 

Any ideas what I can do? 

Thank you,
Wadim

P.S. I apologize for these beginner questions.

**Comment by LekoArts** [comment, marked as answer]
Hi!

> I get on netlify is that "Gatsby requires Node.js 14.15.0 or higher (you have v12.18.0).", which is confusing because I am on version 14.15.4.

Your _local_ Node version (on your computer) is 14.15.4. Netlify runs the build a some Docker image that uses Node 12.18.0. You'll need to set this to >= 14.15.0. See https://docs.netlify.com/configure-builds/manage-dependencies/#node-js-and-javascript

**Reply by wad-im** [reply]
Thank you. That worked of course.
